### ISSUE TITLE
How to evaluate the model in another set?

### ISSUE THREAD

**Issue by vgaraujov** [jbarrow/allennlp_tutorial#6]
Hello @jbarrow, thank you for this fantastic tutorial. I trained the model using the training and validation sets. Now I want to evaluate the model with the test set. I used the following command:

`allennlp evaluate model.tar.gz test.txt --include-package tagging`

However, it returns zero metrics.

```
    "precision-overall": 0.0,
    "recall-overall": 0.0,
    "f1-measure-overall": 0.0,
```

Could you tell me what is the correct way to do that? Thank you so much!
